yarn-project/end-to-end/src/e2e_p2p/p2p_network.ts

```diff
@@ -301,6 +292,24 @@ export class P2PNetworkTest {
     );
   }
 
+  async sendDummyTx() {
+    return await this._sendDummyTx(
+      this.ctx.deployL1ContractsValues.publicClient,
+      this.ctx.deployL1ContractsValues.walletClient,
+    );
+  }
+
+  private async _sendDummyTx(publicClient: PublicClient, walletClient: WalletClient) {
```

Comment:
I would recommend using the l1TxUtils, one some tests you can be unlucky and end up with an underpriced tx on this stuf 💀.